angular/src/lib/tooltip/tooltip.component.ts

```diff
@@ -1,24 +1,22 @@
-import {
-  Component,
-  Input,
-  TemplateRef,
-  OnInit,
-  ViewContainerRef,
-  HostBinding
-} from '@angular/core';
+import { Component, HostBinding, Input, OnInit, TemplateRef, ViewContainerRef } from '@angular/core';
 
 @Component({
   selector: 'md-tooltip',
   styleUrls: ['tooltip.scss'],
   template: `
   <div id="{{id}}" [style.max-width.px]="maxWidth" class="md-tooltip__wrapper">
-    <span *ngIf="text != ''" class="md-tooltip__text" role="tooltip">{{ text }}</span>
+    <span *ngIf="(innerHTML !== '')" class="md-tooltip__text" role="tooltip" [innerHTML]="innerHTML">
+    </span>
+    <span *ngIf="(text !== '')" class="md-tooltip__text" role="tooltip">
+      {{ text }}
+    </span>
```

Comment:
Since I had to separate these (innerHTML will override anything, even if it's undefined), I figured it wouldn't hurt to allow both to operate together.